### PR TITLE
BZ-1283109 - REST API roles restrictions do not work on WebSphere and…

### PR DESCRIPTION
**Pull request by mswiderski**
… WebLogic

@mrietveld please have a look

### CODE REVIEW

File: kie-wb/kie-wb-distribution-wars/src/main/assembly/weblogic12/WEB-INF/web-exec-server.xml [hunk 1]
```diff
@@ -32,11 +32,20 @@
...
+    </init-param>
   </servlet>
   <servlet-mapping>
     <servlet-name>RestServlet</servlet-name>
     <url-pattern>/rest/*</url-pattern>
   </servlet-mapping>
+  <servlet-mapping>
+    <servlet-name>RestServlet</servlet-name>
+    <url-pattern>/rest/*</url-pattern>
+  </servlet-mapping>
```
Comment (mrietveld) [inline kie-wb/kie-wb-distribution-wars/src/main/assembly/weblogic12/WEB-INF/web-exec-server.xml]: It looks like this duplicates lines 41-44.

Comment (mswiderski) [inline kie-wb/kie-wb-distribution-wars/src/main/assembly/weblogic12/WEB-INF/web-exec-server.xml]: correct, will remove it

**Comment by mrietveld** [comment]
+1, although this more a guess than anything else, since I have not tested this myself against WAS or Weblogic. 

Thank you for picking this up and fixing it, @mswiderski.

**Comment by mswiderski** [comment]
closing this as it was decided to not merge it in for now

**Pull request closed without merging**